### DISCUSSION TITLE
Enabled/Disable Taskbar Icon

### DISCUSSION THREAD

**Discussion by howroyd** [howroyd/simonsays#49] [Ideas]
Requested by [MorbidMorke](https://www.twitch.tv/morbidmorke):

Is it possible for the taskbar icon to change when enabled vs disable for those who dont have enough monitors for a dedicated one.

**Comment by honestlyhlf** [comment]
I've not had chance to go through to code thoroughly yet - let alone actually test this - but just looking at gui.py briefly, could adding "enabled_icon.png" and "disabled_icon.png" into src\simonsays_drgreengiant\assets, and the addition of the following lines in gui.py help achieve this?

- Lines 480-481
- Line 486
- Line 493
- Lines 499-508
- Line 513

![enabled_icon](https://github.com/howroyd/simonsays/assets/144053387/bd192837-2a70-425d-b139-83d77e9ed76b)
![disabled_icon](https://github.com/howroyd/simonsays/assets/144053387/6098ede0-3a55-4d09-b0ee-0cebc00f3d78)
[gui.txt](https://github.com/howroyd/simonsays/files/12850655/gui.txt)

**Comment by howroyd** [comment]
@honestlyhlf this looks amazing, thank you so much.  I am planning on spending at least an afternoon or even full day on development this weekend so I shall do that then.

Side note; if you would like we can sort it out so your GitHub account gets credit for the contribution.  If you wanted to I would be more than happy to guide you thought that.  Not sure how GitHub savvy you are so apologies if I am teaching you to suck eggs!

**Comment by honestlyhlf** [comment]
Yes, I'll drop you a message tomorrow @howroyd! A streamer friend has been wanting something like this for Escape from Tarkov (with added features), so I'm keen to get to grips with it and lend a hand (if/where possible, given that I'm only learning). I'm also a complete GitHub noob - I got it mostly so I could keep track of changes on my first project - so please feel free to tell me to stop being a noob and do things properly, haha.
